src/include/compat.h

```diff
@@ -21,6 +21,11 @@
 #define PROCPREFIX
 #endif
 
+#include <fcntl.h>
+#ifdef F_OFD_SETLK
```

Comment:
`ifndef`.

Reply:
@tchaikov 
testbuilder has a dead PSU, got to fix that first.